OpenOversight/app/utils.py

```diff
@@ -189,6 +189,17 @@ def allowed_file(filename):
            filename.rsplit('.', 1)[1].lower() in current_app.config['ALLOWED_EXTENSIONS']
 
 
+def requires_conversion(filename):
+    return '.' in filename and \
+        filename.split('.', 1)[1].lower() in current_app.config['PHOTO_REQUIRES_CONV']
+
+
+def convert_image(filename):
```

Comment:
does this _convert_ e.g. tiff and psd files to jpegs or just rename them? we can use imagemagick for the conversion, i think it should work for both tiff and psds (though we should test that, each are good candidates for unit tests)

Reply:
Right, so I didn't realize imagemagick had a python binding? Should I just use PIL? 

(So there is no way to avoid the os.open/decompression bomb issue?)

Reply:
oh right right, Pillow (Python 3 version of PIL) should do the trick! there's also a decent imagemagick API called `wand` if you find that Pillow is lacking support for a filetype that you want to support.

But yeah I'm not seeing a way to convert these images without using something like Pillow (short of using a third-party service).

Reply:
According to the Pillow docs for this method (https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.save),

"Saves this image under the given filename. If no format is specified, the format to use is determined from the filename extension, if possible."

So it would appear this is actually converting the files to the new format, not just changing the name. 

However, maybe we should have a test for this if we don't already?